tools/gmsh2nek/gmsh2nek.f90

```diff
@@ -1934,17 +1941,20 @@ subroutine open_re2
       use SIZE
 
       character*80  hdr
-
+      integer nBCre2
 
       real*4 test
       data   test  / 6.54321 /
 
       call byte_open(re2name,ierr)
-            
+
+      nBCre2 = 1
+      if (num_elem.ne.eftot)  nBCre2  = 2
+
 !  Write the header
       call blank     (hdr,80)    
-      write(hdr,1) num_elem, num_dim, eftot
-    1 format('#v002',i9,i3,i9,' this is the hdr')
+      write(hdr,1) num_elem, num_dim, eftot, nBCre2
+    1 format('#v004',i9,i3,i9,i3,' this is the hdr')
```

Comment:
I don't think the format matches the one we're using for reading.

Reply:
fixed